recipes-kernel/linux/linux-intel-acrn_4.19.inc

```diff
@@ -23,4 +23,5 @@ KERNEL_FEATURES_append = " features/netfilter/netfilter.scc \
                            features/security/security.scc  \
                            cfg/hv-guest.scc \
                            cfg/paravirt_kvm.scc \
+                           features/net/stmicro/stmmac.scc \
```

Comment:
Can you please change features/net/stmicro/stmmac.scc -> features/net/stmicro/stmmac.cfg in your patch at all the places !

Reply:
OK